deps/build.jl

```diff
@@ -17,6 +17,7 @@ gapdirname = content[1]
 
 # the location the GAP source code is installed
 gap_src_root = abspath(joinpath(artifact_dir, "$(gapdirname)"))
+gap_src_root = ENV["GAPROOT"]
```

Comment:
That shouldn't stay in there, at least not in this form.

Reply:
That was testing code that I accidentally committed. Removed.